File: WebApp/Scripts/app/model/TC_PURCH_M.js
Ext.define('WEBAPP.model.TC_PURCH_M', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'PUR_NO', type: 'string' },
        { name: 'PUR_DATE', type: 'string' },
        { name: 'TC_TYPE', type: 'string' },
        { name: 'PUR_UNM', type: 'string' },
        { name: 'PURCH_ST', type: 'string' },
        { name: 'PUR_NOTE', type: 'string' },


        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/TC/TC_PURCH_M.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class TC_PURCH_M
    {
        public string PUR_NO { get; set; }
        public string PUR_DATE { get; set; }
        public string TC_TYPE { get; set; }
        public string PUR_UNM { get; set; }
        public string PURCH_ST { get; set; }
        public string PUR_NOTE { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }


        public string TC_TYPE_NAME { get; set; }
        public string PUR_DATE_DISPLAY { get; set; }
        public string PUR_UNM_NAME { get; set; }
    }
}